### PR TITLE
feat: Add bool for ABAC

### PR DESCRIPTION
**Pull request by 00LT00**
Signed-off-by: 00LT00 <lightning@zerokirin.online>
#603

### CODE REVIEW

**Comment by 00LT00** [comment]
plz review @hsluoyz @lucas-57blocks  @nodece

**Review by GopherJ — CHANGES_REQUESTED** [review]
Please iterate only one time instead

**Comment by lucas-57blocks** [comment]
> Please iterate only one time instead if we insist on this solution.
> 
> I personally don't like this solution, it's users' responsibility to ensure expression being given in policy.

I couldn't agree more.

**Comment by nodece** [comment]
The feature is compatible RBAC in ABAC: #583

File: enforcer.go [hunk 1]
```diff
@@ -446,7 +446,7 @@ func (e *Enforcer) enforce(matcher string, explains *[]string, rvals ...interfac
 				for _, ruleName := range ruleNames {
 					if j, ok := parameters.pTokens[ruleName]; ok {
 						rule := util.EscapeAssertion(pvals[j])
-						if strings.Contains(rule, ">") || strings.Contains(rule, "<") || strings.Contains(rule, "=") {
+						if strings.Contains(rule, ">") || strings.Contains(rule, "<") || strings.Contains(rule, "=") || strings.Contains(rule, "true") {
```
Comment (nodece) [inline enforcer.go:449]: miss `false`.

**Comment by nodece** [comment]
@00LT00 It is better to use `fix`.

File: enforcer.go (hunk 1 above)
Comment (00LT00) [reply enforcer.go:449]: `false` will return "false" directly through the other branches. I think it is meaningless.

**Comment by hsluoyz** [comment]
What does this PR mean?

**Comment by 00LT00** [comment]
just for the #603 and #583

**Comment by nodece** [comment]
#583 has been rollback, so closing the PR.

**Pull request closed without merging**